torch/utils/cpp_extension.py

```diff
@@ -536,6 +544,14 @@ def win_wrap_ninja_compile(sources,
             if not self.compiler.initialized:
                 self.compiler.initialize()
             output_dir = os.path.abspath(output_dir)
+
+            # Convert relative path in self.compiler.include_dirs to absolute path if any,
+            # For ninja build, the build location is not local, the build happens
+            # in a in script created build folder, relative path lost their correctness.
+            # To be consistent with jit extension, we allow user to enter relative include_dirs
+            # in setuptools.setup, and we convert the relative path to absolute path here
```

Comment:
Instead of duplicating this comment in two places (and then forcing subsequent editors to update it in both places), convert it into a note! In one location, say `Note [Absolute include_dirs]` and then the rest of the note, and then in relevant locations just say `See Note [Absolute include_dirs]`. You should also add a reference to this note at `_to_absolute_path`